test/snippet/core/detail/all_of_type_list_2.cpp

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
remove newline. (check all your snippets)
what is the difference between this file and all_of_type_list.cpp?

Reply:
The call to the function. In version one you can pass a type_list instance as argument in 2 you give an explicit template argument